ProcessLib/RichardsComponentTransport/RichardsComponentTransportFEM-impl.h

```diff
@@ -152,8 +156,9 @@ void LocalAssemblerData<ShapeFunction, IntegrationMethod, GlobalDim>::assemble(
         auto const density = _process_data.fluid_properties->getValue(
             MaterialLib::Fluid::FluidPropertyType::Density, vars);
         auto const& decay_rate = _process_data.decay_rate(t, pos)[0];
-        auto const& molecular_diffusion_coefficient =
-            _process_data.molecular_diffusion_coefficient(t, pos)[0];
+        auto const& diffusion =
```

Comment:
Better to use more describing names: `molecular_diffusion_coefficient`.

Reply:
Reverted parameter naming convention back.